tools/test-proxy/Azure.Sdk.Tools.TestProxy/Common/RecordingType.cs

```diff
@@ -0,0 +1,13 @@
+﻿using System;
```

Comment:
NIT: Remove unused using statements.  You can use the "Run Code Cleanup" tool in VS to do this automatically.